frontend/src/views/web/error-pages/error-404.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html ng-app="evalai">
+<head>
+    <title> 
+      404 | Page Not Found
+    </title>
+  <link rel="stylesheet" type="text/css" href="/dist/vendors/vendor.css">
+
+  <!-- fonts css -->
+  <link rel="stylesheet" type="text/css" href="/dist/css/font-awesome.css">
+  <!-- evalAi main styles -->
+  <link rel="stylesheet" type="text/css" href="/dist/css/main.css">
+
+</head>
+
+<body>
+  <div ui-view>
+
+    <div class="wrapper">
+      <div class="error_container">
+        <div class="error">
```

Comment:
@RishabhJain2018 : 'error' class name is common, so please change the name to some unique name like fof_error, fof_msg etc.